doc/gpu/index.rst

```diff
@@ -86,8 +86,7 @@ Multiple GPUs can be used with the ``gpu_hist`` tree method using the ``n_gpus``
 
 .. note:: Enabling multi-GPU training
 
-  Default installation may not enable multi-GPU training. To use multiple GPUs, make sure to read :ref:`build_gpu_support`.
-XGBoost supports multi-GPU training on a single machine via specifying the `n_gpus' parameter.
+  Default installation may not enable multi-GPU training. To use multiple GPUs, make sure to read :ref:`build_gpu_support`.  XGBoost supports multi-GPU training on a single machine via specifying the `n_gpus' parameter.
```

Comment:
`XGBoost supports multi-GPU training on a single machine via specifying the `n_gpus' parameter.` This sentence can be removed as this is no longer true.